From 0ae9ca1e3cea8135c3ab8064d846f3d836497fa7 Mon Sep 17 00:00:00 2001
From: David Steele <dsteele@gmail.com>
Date: Tue, 8 Oct 2013 19:08:24 -0400
Subject: [PATCH 16/20] One more possible uncaught exception (not seen in the
 wild).

in XmppConnection _write_socket().
(cherry picked from commit 603ee43ff179f130d53cf50aba12364133d70ea1)
---
 cloudprint/xmpp.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cloudprint/xmpp.py b/cloudprint/xmpp.py
index 602e7d3..98cb830 100644
--- a/cloudprint/xmpp.py
+++ b/cloudprint/xmpp.py
@@ -70,7 +70,7 @@ class XmppConnection(object):
         try:
             self._wrappedsock.sendall(msg)
             self._nextkeepalive = time.time() + self._keepalive_period
-        except Exception:
+        except:
             self._connected = False
             raise
 
-- 
1.8.4.rc3

